src/constants/colors/index.js

```diff
@@ -2,15 +2,22 @@
  * Note: Each config is exported separately to be rendered in storybook:
  * https://github.com/PocketDerm/PocketDerm/blob/master/resources/assets/component-library/stories/colors/index.js#L5
  */
+import { withDeprecationWarning } from '../../utils';
+
+const DEFAULT = '#706D87';
+const DEFAULT_BACKGROUND = '#ededf0';
+const DEFAULT_BORDER = '#e1e0e6';
 
 export const brandColors = {
   // Primary 1
   purple100: '#332e54',
+  purple85: '#524D6E',
   purple80: '#5C5876',
```

Comment:
you probably want to move this and purple 60 directly into the `legacyColors` object rather than referencing this `brandColors.purple80` from the legacy colors.

Reply:
I did that but later realized legacyColor was not exported.